indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -63,17 +81,42 @@ export default function EditableList() {
     setSortedList(_sortList({sortBy, sortDirection}));
   };
 
+  const renderCode = code => code || Translate.string('n/a');
+  const renderEditor = editable => {
+    !_.isEmpty(editable) ?? (<Icon name="user outline" /> && editable.editor.fullName);
```

Comment:
This line looks really weird. Why do we need null-coalescing here (does our babel config transpile this properly?), and the `&&` seems strange since the left side is never falsy...

Reply:
null-coalescing worked here but the the editor name was not visible because of `&&`. Changes that.

Reply:
but why would `_.isEmpty(editable)` ever return null?